Resources/views/CRUD/edit_orm_one_to_many.html.twig

```diff
@@ -120,10 +144,20 @@ file that was distributed with this source code.
 
         {% if sonata_admin.edit == 'inline' %}
 
-            {% if sonata_admin.field_description.associationadmin.hasroute('create') and sonata_admin.field_description.associationadmin.isGranted('CREATE') and btn_add and ( sonata_admin.field_description.options.limit is not defined or form.children|length < sonata_admin.field_description.options.limit ) %}
+            {% if sonata_admin.field_description.associationadmin.hasroute('create')
+                and sonata_admin.field_description.associationadmin.isGranted('CREATE')
+                and btn_add
+                and (
+                    sonata_admin.field_description.options.limit is not defined or
+                    form.children|length < sonata_admin.field_description.options.limit
+                )
+            %}
```

Comment:
@core23 : this one was crazy… I don't know people are afraid of linebreaks, especially inside a twig tag.